### ISSUE TITLE
Use base64 feature

### ISSUE THREAD

**Issue by Kixunil** [Kixunil/payjoin#1]
Same shit as in https://github.com/Kixunil/loptos/issues/1

**Comment by DanGould** [comment]
Seems like the blocking issue is merged now. Also, wouldn't this help fix`load_psbt_from_base64` from blocking and breaking "Additional advantage is it doesn't care whether you use async, blocking, tokio, sync-std hyper, actix or whatever. " since it accepts `impl std::io::Read` as param?

**Comment by Kixunil** [comment]
Yeah, `std::io::Read` is not a problem because you can simply supply pre-read `&mut &[u8]` (but sadly, this seems to be less common knowledge). I believe that was the intended way of doing it if you wanted to do non-blocking read. Ideally both libraries would natively support non-blocking reads but that will take some time to implement.

**Comment by DanGould** [comment]
I made some mistakes passing the response body to process_response when using `reqwest::Response` vs `reqwest::blocking::Response` since that &mut &[u8] prep from serialized base64 was not clear to me.

Other than a comment in docs, I'm not sure how to make the interface robust against these integration errors since `process_response` just takes bytes.

**Comment by Kixunil** [comment]
Oh, good point! Maybe the method should have `_blocking` suffix.
